File: xFit_rabbit/xFit_rabbit/Data/Narudzba.cs
using System;
using System.Collections.Generic;

namespace xFit_rabbit.Data;

public partial class Narudzba
{
    public int NarudzbaId { get; set; }

    public string? BrojNarudzbe { get; set; }

    public DateTime? Datum { get; set; }

    public string? Status { get; set; }

    public double? Iznos { get; set; }

    public int? KorisnikId { get; set; }

    public virtual Korisnik? Korisnik { get; set; }

    public virtual ICollection<StavkaNarudzbe> StavkaNarudzbes { get; } = new List<StavkaNarudzbe>();

    public virtual ICollection<Transakcija> Transakcijas { get; } = new List<Transakcija>();
}

File: xFit/UI/xfit_admin/lib/models/narudzba.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:xfit_admin/models/korisnik.dart';

part 'narudzba.g.dart';

@JsonSerializable()
class Narudzba{
	int? narudzbaId;
  String? brojNarudzbe; 
  String? status; 
  DateTime? datum; 
  double? iznos;
  int? korisnikId;
  Korisnik? korisnik;
  

  Narudzba(this.narudzbaId, this.brojNarudzbe, this.status, this.datum, this.iznos,this.korisnikId, [this.korisnik]);

  factory Narudzba.fromJson(Map<String, dynamic> json) => _$NarudzbaFromJson(json);

  Map<String, dynamic> toJson() => _$NarudzbaToJson(this);

}